ac/ac-timedQuiz/src/ActivityRunner.jsx

```diff
@@ -164,13 +164,21 @@ const Question = props => {
 };
 
 const Main = withState('question', 'setQuestion', null)(props => {
-  const { activityData, question, setQuestion, data } = props;
+  const { activityData, question, setQuestion, data, logger } = props;
   const { questions, delay, guidelines } = activityData.config;
   const { name } = props.userInfo;
   let shuffledQ = questionsWithIndex(props);
   if (question === null) {
     shuffledQ = shuffledQuestions(props);
-    const start = () => setQuestion('waiting');
+    const start = () => {
+      setQuestion('waiting');
+      logger([
+        {
+          type: 'progress',
+          value: data.progress / activityData.config.questions.length
+        }
+      ]);
```

Comment:
You created a new 'progress' log that pops up for each question, making it two logs per question. As we have already too many logs, I'd rather use an other solution. As your only goal here is to have at least one log per instance, why not use the `activityDidMount` log common to all instances and write `progress = 0` when this log is received?

Reply:
I don't want to put it on the activity mounting because that can happen before the student actually starts the activity since they can sit on the start screen for a long time. I wanted to get an actual accurate measurement of the time they took on the task, which is why I put it on the start button.

Reply:
OK! I misunderstood the function you updated, I think it's good like this now.